xml/System/String.xml

```diff
@@ -11194,6 +11232,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
    
   
 ## Examples  
+
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]
```

Comment:
you don't need this note here. You have already on top of the page.

```suggestion
```

Reply:
Done.